.github/workflows/dusk_ci.yml

```diff
@@ -26,3 +26,9 @@ jobs:
     uses: dusk-network/.github/.github/workflows/run-tests.yml@main
     with:
       test_flags: --no-default-features 
+  
+  test_serde:
+    name: Serde tests
+    uses: dusk-network/.github/.github/workflows/run-tests.yml@main
+    with:
+      test_flags: --features=serde
```

Comment:
I would not add another test run but add the `serde` feature to the `test_nightly_std` tests